<ui version="4.0" >
 <class>StartQuiz</class>
 <widget class="QDialog" name="StartQuiz" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>start Quiz</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="icons.qrc" >:/icons/icons/help.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Query</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <item>
           <widget class="QRadioButton" name="allRadioButton" >
            <property name="text" >
             <string>&amp;all Vocables</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="onlyNewRadioButton" >
            <property name="text" >
             <string>only &amp;new Vocables</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="onlyExpiredRadioButton" >
            <property name="text" >
             <string>only &amp;expired Vocables</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Direction</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QRadioButton" name="directionNativeForeignRadioButton" >
            <property name="text" >
             <string>Native » Foreign</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="directionForeignNativeRadioButton" >
            <property name="text" >
             <string>Foreign » Native</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="directionRandomRadioButton" >
            <property name="text" >
             <string>Random</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Lessons</string>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>9</number>
        </property>
        <property name="topMargin" >
         <number>9</number>
        </property>
        <property name="rightMargin" >
         <number>9</number>
        </property>
        <property name="bottomMargin" >
         <number>9</number>
        </property>
        <item>
         <widget class="QListWidget" name="lessonsList" >
          <property name="selectionMode" >
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>allRadioButton</tabstop>
  <tabstop>onlyNewRadioButton</tabstop>
  <tabstop>onlyExpiredRadioButton</tabstop>
  <tabstop>lessonsList</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StartQuiz</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>445</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel" >
     <x>437</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
